package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "scripture-resources-rcl",
   "description": "A React Component Library for Rendering Scripture Resources.",
-  "version": "2.1.1",
+  "version": "2.2.0",
```

Comment:
The changes made in this pull request imply a major version change (3.* ?) based on semver because it seems that it isn't backward compatible

Reply:
@mannycolon It looks to me that it is backwards compatible because he is adding an optional options parameter.